rust/libraries/acars_data_processing/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+#[cfg(test)]
+mod tests {
+    #[test]
```

Comment:
this test does not test anything useful

Reply:
Ehehe, it would appear that I created this when mapping things out and then moved the code elsewhere.

It's a completely empty library with a placeholder test, oops...